index.html

```diff
@@ -390,7 +390,7 @@ <h3>
       </section>
       <section>
         <h3>
-          <a>`RemotePlayback`</a> interface
+          {{RemotePlayback}} interface
```

Comment:
If we get back to definitions in the prose, I would actually propose to have the definition target the heading here:

```suggestion
          <dfn>RemotePlayback</dfn> interface
```

Reply:
I'm happy to do that but to be honest am a little confused about how ReSpec treats definitions.   ReSpec already generates a `<dfn>` tag for RemotePlayback from the WebIDL:

```
<dfn data-export="" data-dfn-type="interface" id="dom-remoteplayback" data-idl="interface" data-title="RemotePlayback" data-dfn-for="" class="idlID" tabindex="0" aria-haspopup="dialog"><code>RemotePlayback</code></dfn>
```

How would ReSpec (or the definition exporter for cross-referencing) know which definition to use if there are multiple?

Reply:
There would not be multiple definitions. ReSpec only generates a `<dfn>` tag from the WebIDL if it does not already exist in the prose. It will rather create `<a>` throughout the WebIDL block when the prose contains the relevant definitions.

Reply:
OK, please check the current PR which follows the suggested pattern.